src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -1251,6 +1267,137 @@ static void add_stream_kafka_command(int connFd, std::string &kafka_server_IP, c
     input_stream_handler_thread = thread(&StreamHandler::listen_to_kafka_topic, stream_handler);
 }
 
+void add_stream_hdfs_command(std::string masterIP,int connFd, std::string &hdfs_server_IP, std::thread &input_stream_handler_thread,
```

Comment:
```suggestion
void addStreamHdfsCommand(std::string masterIP,int connFd, std::string &hdfsServerIP, std::thread &inputStreamHandlerThread,
```

Reply:
There is issue I see with function names and variable names. Please follow the approach I have used in my comments.

Reply:
Changed variable names, and method names into camel case notation.